aptos-move/framework/aptos-framework/sources/account/account_abstraction.move

```diff
@@ -295,16 +295,16 @@ module aptos_framework::account_abstraction {
     ) acquires DispatchableAuthenticator {
         let bob_addr = signer::address_of(bob);
         create_account_for_test(bob_addr);
-        assert!(!using_dispatchable_authenticator(bob_addr), 0);
+        assert!(!using_dispatchable_authenticator(bob_addr));
```

Comment:
what does this mean here?

Reply:
assert! has a new version that no longer needs the error code. In tests, error codes are not needed because of stack trace